lang/sv/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: swedish
+title: Semantisk versionshantering 2.0.0
+---
+
+Semantisk versionshantering 2.0.0
+==============================
+
+Sammanfattning
+-------
+
+Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
```

Comment:
Very good suggestion. Making a direct translation that I made is not always the best. But is it not a bit too far from the original text? I'm not saying there is anything wrong, but still want to discuss it. If the text differs too much from the original, it becomes difficult to update when changes occur on the original text.